### ISSUE TITLE
Values for chart indicate wrong configuration for secrets

### ISSUE THREAD

**Issue by boxcee** [stakater/IngressMonitorController#205]
The chart [values.yaml](https://github.com/stakater/IngressMonitorController/blob/ca949f8c1f00aea5523c9470ff3bfa073c181b3e/deployments/kubernetes/chart/ingressmonitorcontroller/values.yaml#L47) suggest to put the secrets into the `configMap`.

```
ingressMonitorController:
  data:
    config.yaml: |-
        secrets: {}
```

It is wrong. Looking at the [deployment.yaml](https://github.com/stakater/IngressMonitorController/blob/ca949f8c1f00aea5523c9470ff3bfa073c181b3e/deployments/kubernetes/chart/ingressmonitorcontroller/templates/deployment.yaml#L54) the secrets are clearly still specified under `ingressMonitorController.secrets`.

**Comment by usamaahmadkhan** [comment]
@boxcee thanks for pointing it out. I'll note it down on my TODO list.

**Comment by usamaahmadkhan** [comment]
fixed in v1.0.87.

**Comment by rasheedamir** [comment]
fixed by #206
